go/libraries/doltcore/sqle/logictest/DoltRegressionsJenkinsfile

```diff
@@ -0,0 +1,91 @@
+                            sh "rm -rf .dolt && dolt init"
+                            sh "rm -rf $CREDSDIR && mkdir $CREDSDIR"
+                            sh "cat $DOLT_CREDS > /home/jenkins/.dolt/creds/5jl65iugbs7olde8n8qqgvujs1bhp8c84bopgi78n26is.jwk"
+                            sh "dolt config --global --add user.creds 5jl65iugbs7olde8n8qqgvujs1bhp8c84bopgi78n26is"
+                            sh "dolt config --global --add metrics.disabled true"
+                            sh "dolt creds use 5jl65iugbs7olde8n8qqgvujs1bhp8c84bopgi78n26is"
+                            sh "rm -rf temp && mkdir temp"
+                            script {
+                                def j = env.TEST_N_TIMES.toInteger()
+                                for (int i = 1; i <= j; ++i) {
+                                    sh "go run . run ../../../../../../sqllogictest/test/select1.test > temp/results${i}.log"
+                                    sh "go run . parse \"$DOLT_VERSION\" temp/results${i}.log > temp/parsed${i}.json"
+                                }
+                            }
+                        }
+                        dir ("dolt-sql-performance") {
+                            sh "dolt checkout nightly"
+                            script {
+                                def j = env.TEST_N_TIMES.toInteger()
+                                for (int i = 1; i <= j; ++i) {
+                                    sh "dolt table import -u nightly_dolt_results ../go/libraries/doltcore/sqle/logictest/main/temp/parsed${i}.json"
+                                    sh "dolt add nightly_dolt_results"
+                                    sh "dolt commit -m \"update dolt sql performance results\""
+                                }
+                            }
+                            sh "rm -f mean.csv"
+                            sh "dolt sql -r csv -q \'select version, test_file, line_num, avg(duration) as mean_duration, result from dolt_history_nightly_dolt_results where version=\"${DOLT_VERSION}\" group by line_num;\' > mean.csv"
+                            sh "dolt table import -u nightly_dolt_mean_results mean.csv"
+                            sh "dolt add nightly_dolt_mean_results"
+                            sh "dolt commit -m \"update dolt sql performance mean results\""
+                            sh "dolt push origin nightly"
+                            sh "dolt checkout regressions"
+                            sh "dolt merge nightly"
+                            sh "dolt add ."
+                            sh "dolt commit -m \"merge nightly\""
+                            sh "dolt push origin regressions"
+                            script {
+                                env.DURATION_REGRESSIONS = sh(returnStdout: true, script: "dolt sql -q 'select * from releases_nightly_duration_change' -r csv | wc -l | tr -d '[:space:]'")
```

Comment:
Here we query the view to ensure that it only contains a single row, which is the header row... Any other rows returned will indicate a test failure